### PR TITLE
Script to update e2e cluster

### PR DESCRIPTION
**Pull request by yutongz**
**Release note**:

```release-note
none
```
This is a script to create a new e2e cluster for one repo and update the secret in Prow cluster to point to the new cluster credentials.

### CODE REVIEW

File: scripts/update-e2e-cluster.sh [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+CONFIG_BACKUPED=false
+
+# Switch to prow cluster
+gcloud container clusters get-credentials ${PROW_CLUSTER} --zone ${PROW_ZONE} --project ${PROW_PROJECT}
+
+# Update kubeconfig
+SECRET_NAME="${REPO}-e2e-rbac-kubeconfig"
+kubectl delete secret ${SECRET_NAME} -n ${PROW_TEST_NS}
+sleep 5
+kubectl -n test-pods create secret generic ${SECRET_NAME} --from-file=${TEMP_DIR}
```
Comment (sebastienvas) [inline scripts/update-e2e-cluster.sh]: change to PROW_TEST_NS

File: scripts/update-e2e-cluster.sh [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+kubectl get secret -n test-pods
```
Comment (sebastienvas) [inline scripts/update-e2e-cluster.sh]: same here

File: scripts/update-e2e-cluster.sh [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  --machine-type ${MACHINE_TYPE} --num-nodes ${NUM_NODES} --no-enable-legacy-authorization --enable-kubernetes-alpha --quiet \
+  ||  echo 'Failed')
+  [[ ${result} == 'Failed' ]] || break
+  if [ ${i} -eq 2 ]; then
+    echo "Cannot create a rotation cluster for ${REPO}"; exit 1
+  fi
+done
+
+# Keep new config into temp dir and put original config back
+TEMP_DIR="${HOME}/.kube/${REPO}_TEMP"
```
Comment (sebastienvas) [inline scripts/update-e2e-cluster.sh]: you might want to use 
```
TEMP_DIR="$(mktemp -d)"
```
so that you don't polute your .kube directory

**Review by sebastienvas — APPROVED** [review]
Nice - a few change

**Comment by yutongz** [comment]
PTAL

**Review by sebastienvas — APPROVED** [review]

**Pull request merged**